packages/framework/src/Form/Admin/Category/CategoryFormType.php

```diff
@@ -257,4 +261,40 @@ private function getCategoryNameForPlaceholder(DomainConfig $domainConfig, ?Cate
 
         return $category === null ? '' : $category->getName($domainLocale);
     }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     * @param \App\Model\Category\Category|null $category
+     */
+    protected function buildFilterParameters(FormBuilderInterface $builder, ?\App\Model\Category\Category $category): void
+    {
+        if ($category === null) {
+            return;
+        }
+        $parametersFilterBuilder = $builder->add('parametersGroup', GroupType::class, ['label' => t('Filter parameters')]);
+
+        $parameterNamesById = [];
+
+        $parametersUsedByProductsInCategory = $this->parameterRepository->getParametersUsedByProductsInCategory($category, $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID));
+
+        foreach ($parametersUsedByProductsInCategory as $parameter) {
+            $parameterNamesById[$parameter->getId()] = $parameter->getName();
+        }
+
+        $parametersFilterBuilder->add('parametersPosition', SortableValuesType::class, [
```

Comment:
`CategoryData::$parametersPosition` property is still defined in the project-base, it should be moved as well, right?